source/assets/stylesheets/_header.scss

```diff
@@ -117,13 +117,17 @@
     a#proposition-name:focus {
       color: $text-colour;
     }
-    a.menu {
+    .menu {
       @include core-16;
       color: $white;
       display: block;
       float: right;
       text-decoration: none;
-      padding-top: 6px;
+      font-weight: bold;
+      padding: 2px 6px 1px;
+      font-size: 13px !important;
+      background-color: $black;
+      border-radius: 3px;
```

Comment:
in relation on the comment above, can we not introduce new visual to the button/a tag?
if we're changing it to a button then let's only add styles that make it look like it did before. 
no border radius please :)

Reply:
So the idea Oliver had was to make the link look like the developer docs, but actually it's a smaller change just to keep the link and change it's role to button for a better VoiceOver.

Reply:
So back to a link and no visual change :-)

Reply:
thumbs 👍